.gitignore

```diff
@@ -30,4 +30,6 @@ Dockerfile.cross
 # tmp directories built for e2e tests
 tests/e2e/local
 tests/kustomize-e2e/local
-tests/helm-e2e/local
\ No newline at end of file
+tests/helm-e2e/local
+
+config/crd/bases/_.yaml
```

Comment:
good solution for now at least :)

Reply:
I spent a bit of time investigating why that is being generated, but could not figure it out. So I decided to exclude it at least for now.